### PR TITLE
Add 3 decimal places to LiveSplit.DetailedTimer + code cleanup

### PR DESCRIPTION
**Pull request by 10-days-till-xmas**
Related to https://github.com/LiveSplit/LiveSplit/issues/930

### CODE REVIEW

**Comment by just-ero** [comment]
I would personally prefer if you split this into multiple PRs, or at the very least into multiple commits.

**Comment by 10-days-till-xmas** [comment]
i could annotate it instead for now?

**Comment by just-ero** [comment]
It'll be fine, just something for the future. :) Makes it easier to grasp the changes.

File: src/LiveSplit.DetailedTimer/UI/Components/DetailedTimerSettings.Designer.cs [hunk 1]
```diff
@@ -1094,20 +1187,22 @@ private void InitializeComponent()
...
-            this.AutoScaleDimensions = new System.Drawing.SizeF(6F, 13F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(8F, 16F);
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
             this.Controls.Add(this.tableLayoutPanel1);
+            this.Margin = new System.Windows.Forms.Padding(4);
             this.Name = "DetailedTimerSettings";
-            this.Padding = new System.Windows.Forms.Padding(7);
-            this.Size = new System.Drawing.Size(459, 991);
+            this.Padding = new System.Windows.Forms.Padding(9);
+            this.Size = new System.Drawing.Size(612, 1220);
```
Comment (wooferzfg) [inline src/LiveSplit.DetailedTimer/UI/Components/DetailedTimerSettings.Designer.cs:1205]: Did you forget to change the scaling back to 100% when working on this component?

**Pull request merged**